src/Storage/DatabaseStorage.php

```diff
@@ -351,6 +352,17 @@ protected function preaggregateAverages(Collection $entries): Collection
         ]);
     }
 
+    /**
+     * Collapse values.
+     *
+     * @param  \Illuminate\Support\Collection<int, \Laravel\Pulse\Value>  $values
+     * @return \Illuminate\Support\Collection<int, \Laravel\Pulse\Value>
+     */
+    protected function collapseValues(Collection $values): Collection
+    {
+        return $values->reverse()->unique(fn (Value $value) => [$value->key, $value->type]);
+    }
+
```

Comment:
This was added because one of my tests failed in Postgres 14.

See https://github.com/laravel/pulse/pull/250/files#diff-82b9572841700c12fb8fdbbc144371ae2af52f68529f5b0cbccc1acc3910fb10R428-R446